src/Nri/Ui/Block/V6.elm

```diff
@@ -23,7 +24,7 @@ module Nri.Ui.Block.V6 exposing
 
 ## Patch changes
 
-    Add renderReadAloud
+    Add border styles `dashed` and `underline`
```

Comment:
Yeah, I wasn't sure either. There was no history in this file of a patch change until the one I replaced, and I didn't see anything explicit about it in the documentation on writing PRs.

Reply:
The intention is to provide devs with a quick reference for identifying changes from the last version, either by reading the comment or the docs at https://package.elm-lang.org/packages/NoRedInk/noredink-ui/latest. If one change replaces another in the same patch, it would make sense to update this in the list (by replacing the old change log item). If it doesn't replace the other change, we should append it instead.